From d7458a7415657de9f97b2bb76ef3d5d65bea78ca Mon Sep 17 00:00:00 2001
From: Pablo Neira Ayuso <pablo@netfilter.org>
Date: Fri, 7 Jun 2013 15:01:04 +0200
Subject: include: use C++ headers in C++ mode

Reported-by: Thomas Jarosch <thomas.jarosch@intra2net.com>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

diff --git a/include/libnetfilter_acct/libnetfilter_acct.h b/include/libnetfilter_acct/libnetfilter_acct.h
index 18f90c4..421e15a 100644
--- a/include/libnetfilter_acct/libnetfilter_acct.h
+++ b/include/libnetfilter_acct/libnetfilter_acct.h
@@ -1,10 +1,15 @@
 #ifndef _LIBNETFILTER_ACCT_H_
 #define _LIBNETFILTER_ACCT_H_
 
-#include <sys/types.h>
 #include <stdint.h>
 #include <linux/netfilter/nfnetlink_acct.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sys/types.h>
+
 struct nfacct;
 
 enum nfacct_attr_type {
@@ -39,4 +44,8 @@ int nfacct_nlmsg_parse_payload(const struct nlmsghdr *nlh, struct nfacct *nfacct
 
 int nfacct_snprintf(char *buf, size_t size, struct nfacct *nfacct, uint16_t type, uint16_t flags);
 
+#ifdef __cplusplus
+} /* extern "C" */
+#endif
+
 #endif
-- 
cgit v0.10.2

